Vagrantfile.example

```diff
@@ -37,6 +38,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.provision "ansible" do |ansible|
+      # ansible.playbook = "ansible/configure-rhel-7.yml"
       ansible.playbook = "ansible/pulp-from-source.yml"
```

Comment:
It's a bit unclear (to me at least) how to use CentOS. Do I uncomment line 41, do I need to also comment out line 42, etc.

It would be kind of nice if users could just change the value for `config.vm.box` to `centos/7`. Alternatively, adding instructions to README would be good.

Reply:
It might be better for me to just remove the commented lines.  actually using centos (or rhel) is multi-step, you ave to run with playbook as configure-rhel-7 and then re-provision with the playbook as pulp-from-source.  It's mostly something that QE could use, or devs on very rare occasions.

Reply:
Unused code is broken code. A good starting point would be to make the [`configure-rhel-7.yml`](https://github.com/pulp/devel/blob/master/ansible/configure-rhel-7.yml) playbook smarter, such that it only calls the `configure_rhel_7` role when the target host is a CentOS or RHEL 7 VM. Something like this might do it:

```yaml
# Pseudo-code!
roles:
  - name: configure_rhel_7
    when: host.ansible_distribution == 'RedHat'
```

This means that the user doesn't have to worry about un-commenting multiple lines. They just change `config.vm.box` and are done with it.